packages/expo-file-system/android/src/main/java/expo/modules/filesystem/next/FileSystemDirectory.kt

```diff
@@ -20,10 +20,21 @@ class FileSystemDirectory(file: File) : FileSystemPath(file) {
     return file.isDirectory
   }
 
-  fun create() {
+  fun create(options: CreateOptions = CreateOptions()) {
```

Comment:
```suggestion
  fun create(options: CreateOptions) {
```
You're already passing the default value in the module DSL.

Reply:
yep, but I'm also using it from other functions so I'd like the default to also exist there